src/Helmet.js

```diff
@@ -187,7 +187,11 @@ const updateHtmlAttributes = (attributes) => {
         htmlTag.removeAttribute(attributesToRemove[i]);
     }
 
-    htmlTag.setAttribute(HELMET_ATTRIBUTE, helmetAttributes.join(","));
+    if (helmetAttributes.length === attributesToRemove.length) {
+        htmlTag.removeAttribute(HELMET_ATTRIBUTE);
```

Comment:
If the `HELMET_ATTRIBUTE` tag is removed, won't we lose track of what attributes we need to track?